### DISCUSSION TITLE
Scalable module. Why is it different from what is displayed in the document? It's not together, but separated and separated

### DISCUSSION THREAD

**Discussion by HVGNc** [GitbookIO/community#525] [2. Published content] [answered]
Scalable module. Why is it different from what is displayed in the document? It's not together, but separated and separated
![2](https://github.com/GitbookIO/community/assets/87613584/0d0b81c2-0941-4af0-a683-6cf2d9ec787e)
![3](https://github.com/GitbookIO/community/assets/87613584/e8e00b48-122b-4fb5-a045-049f89238b4c)
![4](https://github.com/GitbookIO/community/assets/87613584/c93c987d-828e-49e7-adc6-8034b5291f57)
![5](https://github.com/GitbookIO/community/assets/87613584/20ec99c7-1a13-48d4-a376-13de787ecfbd)

**Comment by HVGNc** [comment]
Scalable module. Why is it different from what is displayed in the document? It's not together, but separated and separated
The ones glued together as shown in the picture are displayed in official documents, but I added them myself and they are all separated in the middle, not continuous

**Reply by thebiglabasky** [reply]
Hi
This is indeed an improvement we made on published docs sites that is not yet reflected in the GitBook editor. 
Are there cases where you would need one or the other or would it be fine to stick to one pattern only?

**Reply by zhimaj** [reply]
Ah。 When can it be applied to the editor? Because the official help demonstration webpage is the kind that can be stacked together, which looks more beautiful and helps with some textual step instructions. At present, it is in an empty and open state, with too much content that looks unattractive, lacking continuity, and taking up a lot of space

**Reply by thebiglabasky** [reply]
We don't have plans on that just yet, so probably in the coming few months. We'll keep this issue posted!

**Reply by HVGNc** [reply]
Ah。 But a long time ago, both the old and new versions of the official document knowledge base were stacked? Why do users actually not have this feature.

**Reply by HVGNc** [reply]
![Uploading 11.png…]()

**Reply by thebiglabasky** [reply, marked as answer]
No we used to have all expandables separated like they show today in the editor, but have worked on improving our published content rendering, where the stacked view was tried. We think that if it works better we should make this also a permanent change in the editor but that's what I don't have a timeline for at the moment
